.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+### Description of Changes
+Please briefly explain the changes here.
+
+### Types of Changes
```

Comment:
Since our jira issues all have a type, I think we could drop this section. Everything else looks good to me. Thanks for putting it together!

Reply:
Thanks for the feedback! I have updated the template.